### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by BahirHakimy**
Implemented the first part of the exercise as described [Here](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects_v1_1.md)
- Added functionality to add new book
- Added functionality to remove a book
- Added functionality to preserve the data in local storage

### CODE REVIEW

File: collection.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+window.onload = render();
+
+const title = document.getElementById('newTitle');
+const author = document.getElementById('newAuthor');
+const addBtn = document.getElementById('add');
+
+addBtn.addEventListener('click', () => {
+  booksArr.push({ title: title.value, author: author.value });
+  saveToLocalStorage();
```
Comment (omar-muhamad) [inline collection.js]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20function%20to%20add%20a%20new%20book%20to%20the%20collection%2C%20with%20title%20and%20author.), Please Create a separate function to add a new book to the collection, with title and author and you can call it here in the event listener.

File: collection.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  list.innerHTML = booksArr.map((book) => (`<li>
+            <article>
+                <p id="title">${book.title}</p>
+                <p id="author">${book.author}</p>
+                <button class="remove">Remove</button>
+            </article>`)).join('');
+
+  const remove = Array.from(document.getElementsByClassName('remove'));
+  remove.forEach((btn, i) => btn.addEventListener('click', () => {
+    booksArr = booksArr.filter((book, ind) => i !== ind);
```
Comment (omar-muhamad) [inline collection.js]: - [ ] Same here, as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20function%20to%20remove%20a%20book%20from%20the%20collection%20(hint%3A%20you%20can%20use%20the%20array%20filter()%20method).), Please create a separate function for removing a book from a list of books and you can call it where ever you need.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @BahirHakimy and @astroboyReloaded,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please add your collaborator @BahirHakimy name and social links to the `authors` section in the README file to make it more professional.

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: collection.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  return cachedBooks ? JSON.parse(cachedBooks) : [];
+};
+
+let booksArr = getFromLocalStorage();
+
+const saveToLocalStorage = () => {
+  window.localStorage.setItem(LOCAL_STORAGE_KEY, JSON.stringify(booksArr));
+};
+
+const deleteBook = (i) => {
```
Comment (anagudelogu) [inline collection.js:15]: - [ ] **[OPTIONAL]** I would suggest that you use meaningful variable names, in this case, `i` doesn't provide a lot of information about what you are expecting as a parameter. 🤓 🔧

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @astroboyReloaded, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**